app/pods/publications/publication/translations/documents/route.js

```diff
@@ -23,7 +23,7 @@ export default class PublicationsPublicationTranslationsDocumentRoute extends Ro
     });
 
     const generatedPiecesQuery = this.store.query('piece', {
-      'filter[translation-activity-generated-by][:id:]': this.translationSubcase.id,
+      'filter[translation-activity-generated-by][subcase][:id:]': this.translationSubcase.id,
```

Comment:
Deze lijkt dan wel juist, die `translation-activity-generated-by` wees naar de `translationActivity`, en we moeten idd subcase erbij insteken als we `translationSubcase` willen hebben